### DISCUSSION TITLE
Welcome to PyChatGPT Discussions!

### DISCUSSION THREAD

**Discussion by rawandahmad698** [rawandahmad698/PyChatGPT#94] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by codeapplejam** [comment]
cant find a way to run it .
installed toml-cli but still nothing

**Reply by rawandahmad698** [reply]
run `pip3 install chatgptpy --upgrade` then use one of the code snippets on readme.md

**Reply by DrMemoryFish** [reply]
> run `pip3 install chatgptpy --upgrade` then use one of the code snippets on readme.md

are there more in-depth instructions on how to set it up, it would be nice to have a user-friendly interface. even if it's rough. with buttons and such, rather than having to keep using the code snippets

**Reply by rawandahmad698** [reply]
> > run `pip3 install chatgptpy --upgrade` then use one of the code snippets on readme.md
> 
> 
> 
> are there more in-depth instructions on how to set it up, it would be nice to have a user-friendly interface. even if it's rough. with buttons and such, rather than having to keep using the code snippets 

There can be. But frankly I'm very busy for such things. Although it's make it even cooler. But I maintain a list of other repos. PR's welcome tho

**Comment by ElMahdiAboulmanadel** [comment]
Can you please share a new discord invite link? The actual one in the README.MD is expired and I'm trying to join the Discord server. Thanks!

**Comment by Pratik3207** [comment]
Hard work
